docs/cli-reference/gardenadm/gardenadm_init.md

```diff
@@ -15,13 +15,17 @@ gardenadm init [flags]
 ```
 # Bootstrap the first control plane node
 gardenadm init --config-dir /path/to/manifests
+
+# Bootstrap the first control plane node using the bootstrap etcd
+gardenadm init --config-dir /path/to/manifests --use-bootstrap-etcd
```

Comment:
```suggestion
```

Let's not advertise this too prominently in the examples - the flag is visible (with its "doc string" in the command help), that's perhaps already enough